net.labymod.LabyMod.yml

```
@@ -0,0 +1,38 @@
+id: net.labymod.LabyMod
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: labymodlauncher
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --env=ELECTRON_DISABLE_SANDBOX='1'
+  - --persist=.minecraft
+modules:
+  - name: LabyMod
+    buildsystem: simple
+    build-commands:
+       - mkdir -p /app/tmp-install/data
+       - bsdtar -xf laby.deb -C /app/tmp-install
+       - bsdtar -xf /app/tmp-install/data.tar.xz -C /app/tmp-install/data
+       - mv /app/tmp-install/data/usr/lib/labymodlauncher/* /app/bin
+       - rm -r /app/tmp-install laby.deb
```

```suggestion
       - bsdtar -xf laby.deb
       - bsdtar -xf data.tar.xz -C data
       - mv data/usr/lib/labymodlauncher/* /app/bin
```

why doing it the simple way when there is a much more complicated way.